stdlib/public/core/SetBridging.swift

```diff
@@ -362,7 +362,7 @@ extension _CocoaSet: _SetBuffer {
   internal func formIndex(after index: inout Index) {
```

Comment:
`formIndex(after:)` is not exposed in `Dictionary` nor `Set`; only the `index(after:)` variants ever get called.

Reply:
And the `@_effects` attribute obviously doesn't apply on it.